### ISSUE TITLE
No action is performed upon invoking the 'Hamburger Menu' button, present in Chat Box while navigating using keyboard tab key/Mouse.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5102]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

All users will be impacted if no action is performed after invoking menu item as users will be blocked and cannot navigate to menu item.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### What is the public URL for the website?

https://microsoft.sharepoint.com/teams/MCAPSAcademy/SitePages/Request-a-Contributor---POC.aspx?xsdata=MDV8MDF8ZGJoYWt0YUBtaWNyb3NvZnQuY29tfGUzYWVmOTc1NDViMTQzYTg4Y2ZhMDhkYmM1YmQ5YjI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2MzgzMjExOTAzODMzODc3NDF8VW5rbm93bnxUV0ZwYkdac2IzZDhleUpXSWpvaU1DNHdMakF3TURBaUxDSlFJam9pVjJsdU16SWlMQ0pCVGlJNklrMWhhV3dpTENKWFZDSTZNbjA9fDMwMDB8fHw%3D&sdata=SSs2MzJWZWN5M2JyTVZGeWp4c0VjbkJlNHIvZEpueXNreC9IZktzSkMxTT0%3D&OR=Teams-HL&CT=1696587959401&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzA5MTQxNzUwNSIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D

### How to reproduce the issue?

Step 1: Open the above URL in New Edge/Chrome browsers.
Step 2: Request Contributors - POC Home page will be appeared. 
Step 3: Navigate to the 'Got questions?' in the page now Ask Iris Now! link is invoked it, chat box appeared. 
Step 4: Navigate to the Menu Button.
Step 5: Verify whether No action is performed upon invoking the 'Menu' button, presented in Chat Box while navigating using keyboard tab key or not.

### What do you expect?

Hamburger menu should be expanded upon invoking the 'Menu' button, presented in Chat Box while navigating using keyboard tab key.

### What actually happened?

No action is performed upon invoking the 'Menu' button, present in Chat Box while navigating using keyboard tab key.
Observation: Keyboard focus is moving to hamburger Menu, but no action is performed upon invoking enter key/mouse.

### Do you have any screenshots or recordings to repro the issue?

<img width="935" alt="MAS 2 1 1-No action is performed upon invoking the 'Menu' button, presented in Chat Box while navigating using keyboard tab key" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/2651a455-d843-4787-a649-79c8670279fd">

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

**Comment by Kirank915** [comment]
#functional;#A11ySev1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug copied from Iris BOT: [ISSUE 1767709](https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1767709)

**Comment by compulim** [comment]
This is out of scope of Web Chat. The title/menu bar is done by Iris Bot and is not part of Web Chat.

**Comment by Kirank915** [comment]
As per the above comment, this issue is not under bot framework team. Hence closed the bug and partner team (WWL) also changed the ownership in bugs.
